monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -216,6 +227,8 @@ class ReportPageComponent extends AuthComponent {
   }
 
   generateReportOverviewSection() {
+    let manual_monkey_hostnames = getManuallyStartedAgents(this.state.agents).map((agent) => getMachineHostname(getAgentMachine(agent, this.state.machines)));
```

Comment:
Hmm, _if_ the call to getAgentMachine ends up returning a null value, then the array could have nulls in it. We may want to filter them out